image-layout.md

```diff
@@ -1,13 +1,13 @@
 ## Open Container Initiative Image Layout Specification
 
-The OCI Image Layout is a slash separated layout of OCI content-addressable blobs and [location-addressable](https://en.wikipedia.org/wiki/Content-addressable_storage#Content-addressed_vs._location-addressed) references (refs).
+The OCI Image Layout is a slash separated layout of OCI content-addressable blobs and [location-addressable](https://en.wikipedia.org/wiki/Content-addressable_storage#Content-addressed_vs._location-addressed) reference tags (tags).
```

Comment:
On Mon, Aug 29, 2016 at 02:07:10PM -0700, Stephen Day wrote:
> > @@ -1,13 +1,13 @@
> >  ## Open Container Initiative Image Layout Specification
> >  
> > -The OCI Image Layout is a slash separated layout of OCI content-addressable blobs and [location-addressable](https://en.wikipedia.org/wiki/Content-addressable_storage#Content-addressed_vs._location-addressed) references (refs).
> > +The OCI Image Layout is a slash separated layout of OCI content-addressable blobs and [location-addressable](https://en.wikipedia.org/wiki/Content-addressable_storage#Content-addressed_vs._location-addressed) reference tags (tags).
>
> Does this mean we are dropping the concept of generic `refs`? I
> thought we were just calling out a "Reserved" structure within
> `refs`?

Ugh.  In order of my decreasing preference:

a. Keep our current generic refs [1].
b. Replace generic refs with refs that have some required tag
   semantics (this PR?).
c. Support *both* generic refs and refs with required tag semantics
   (what you seem to be suggesting with the “Reserved structure”
   comment).

Any type information about refs *should* go in the descriptor media
type field.  I see no reason to bake it into the bucket that holds the
descriptors.

[1]: https://github.com/opencontainers/image-spec/pull/219#issuecomment-243197974